_drafts/autolab-and-docker.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Autolab and Docker"
+author: Mihir Pandya
+
+tags: [autograding, docker, tango]
+---
+
+![]({{site-url}}/assets/docker.svg)
```

Comment:
I'm sorry. It should be `{{ site.baseurl }}`. `{{ site-url }}` only works because when Liquid variables don't exist, they default to `""`.

The purpose of `site.baseurl` is to adjust. If we ever wanted to deploy the blog to a location like `autolabproject.com/blog`, we could simply change `site.baseurl` to `/blog`, and `site.url` to `http://autolabproject.com`.